Sources/Penny/EvolutionChecker.swift

```diff
@@ -162,16 +162,21 @@ actor EvolutionChecker {
 
         let summary = makeSummary(proposal: proposal)
 
+        let upcomingFeatureFlag = proposal.upcomingFeatureFlag.map {
+            "\n**Upcoming Feature Flag:** \($0.flag)"
+        } ?? ""
```

Comment:
Upcoming feature flags are also available in the new format of the evolutions JSON that swift.org returns.